webapp/channels/src/components/youtube_video/youtube_video.tsx

```diff
@@ -71,59 +110,64 @@ export default class YoutubeVideo extends React.PureComponent<Props, State> {
 
         if (this.state.playing) {
             content = (
-                <iframe
-                    src={'https://www.youtube.com/embed/' + videoId + '?autoplay=1&autohide=1&border=0&wmode=opaque&fs=1&enablejsapi=1' + time}
-                    width='480px'
-                    height='360px'
-                    frameBorder='0'
-                    allowFullScreen={true}
-                    allow='accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share'
-                    referrerPolicy={this.props.youtubeReferrerPolicy ? 'strict-origin-when-cross-origin' : 'no-referrer'}
-                    title={videoTitle}
-                    sandbox='allow-scripts allow-same-origin allow-popups allow-presentation'
-                />
+                <div className='video-playing'>
+                    <iframe
+                        src={`https://www.youtube.com/embed/${videoId}?autoplay=1&rel=0${time}`}
+                        title={videoTitle}
+                        width='100%'
+                        height='100%'
+                        frameBorder='0'
+                        allow='accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture'
+                        allowFullScreen={true}
+                        referrerPolicy={this.props.youtubeReferrerPolicy ? 'origin' : undefined}
+                        sandbox='allow-scripts allow-same-origin allow-popups allow-presentation'
+                    />
+                </div>
             );
         } else {
-            const image = metadata?.images[0];
-
             content = (
-                <div className='embed-responsive video-div__placeholder'>
-                    <div className='video-thumbnail__container'>
-                        <ExternalImage src={image?.secure_url || image?.url || ''}>
-                            {(safeUrl) => (
-                                <img
-                                    src={safeUrl}
-                                    alt='youtube video thumbnail'
-                                    className='video-thumbnail'
-                                />
-                            )}
-                        </ExternalImage>
-                        <div className='block'>
-                            <span className='play-button'><span/></span>
-                        </div>
+                <div
+                    className='video-thumbnail__container'
+                    onClick={this.play}
+                    role='button'
+                    aria-label={`Play ${videoTitle} on YouTube`}
+                    tabIndex={0}
+                    onKeyDown={(e) => {
+                        if (e.key === 'Enter' || e.key === ' ') {
+                            e.preventDefault();
+                            this.play();
+                        }
+                    }}
+                >
+                    <img
```

Comment:
This needs to be updated to use `ExternalImage` instead of a raw image tag so that the request for the thumbnail goes through the image proxy if it's enabled on the server. Without that, YouTube would get direct requests from users' browser who just see the thumbnail which is a privacy concern